src/ioctls/vcpu.rs

```diff
@@ -1849,6 +1852,19 @@ impl VcpuFd {
             _ => Err(errno::Error::last()),
         }
     }
+
+    /// If [`Cap::X86BusLockExit`](crate::Cap::X86BusLockExit) was enabled,
+    /// checks whether a bus lock was detected on the last VM exit. This may
+    /// return `true` even if the corresponding exit was not
+    /// [`VcpuExit::X86BusLock`], as a different VM exit may have preempted
+    /// it.
+    ///
+    /// See the API documentation for `KVM_CAP_X86_BUS_LOCK_EXIT`.
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    pub fn bus_lock_detected(&self) -> bool {
```

Comment:
nit: I would prefix the name with `is_` (is_bus_...) to match the naming conventions used for such functions.

Also, can we add a test for it?